manifests/init.pp

```diff
@@ -68,7 +68,7 @@
       $config_client_path_default = '/etc/default/openafs-client'
       $init_script_default        = '/etc/init.d/openafs-client'
       $init_template_default      = 'openafs-client-Ubuntu'
-      $package_name_default       = [ 'openafs-client', 'openafs-doc', 'openafs-krb5', 'openafs-modules-dkms', 'openafs-modules-source', ]
+      $package_name_default       = [ 'openafs-modules-dkms', 'openafs-modules-source', 'openafs-client', 'openafs-doc', 'openafs-krb5', 'dkms' ]
```

Comment:
The special sequence was necessary because the openafs-client is trying to start before the openafs-modules-dkms is installed which will build the needed opens kernel-module.
I have disabled the auto-startup of the new created packages. So now the order is not so important anymore.

/Ralf


Am 23.02.2015 um 09:15 schrieb Phil Friderici <notifications@github.com<mailto:notifications@github.com>>:


In manifests/init.pp<https://github.com/Phil-Friderici/puppet-module-afs/pull/33#discussion_r25148483>:

> @@ -68,7 +68,7 @@
>        $config_client_path_default = '/etc/default/openafs-client'
>        $init_script_default        = '/etc/init.d/openafs-client'
>        $init_template_default      = 'openafs-client-Ubuntu'
> -      $package_name_default       = [ 'openafs-client', 'openafs-doc', 'openafs-krb5', 'openafs-modules-dkms', 'openafs-modules-source', ]
> +      $package_name_default       = [ 'openafs-modules-dkms', 'openafs-modules-source', 'openafs-client', 'openafs-doc', 'openafs-krb5', 'dkms' ]


just fyi: https://docs.puppetlabs.com/learning/ordering.html
If we need a special sequence here we need to code it. But I don't think it neccessary here.

—
Reply to this email directly or view it on GitHub<https://github.com/Phil-Friderici/puppet-module-afs/pull/33/files#r25148483>.